src/lib/JreSetupManager.ts

```diff
@@ -3,7 +3,7 @@ import {AsyncCreatable} from '@salesforce/kit';
 import {Controller} from '../Controller';
 
 import process = require('process');
-import * as findJavaHome from 'find-java-home';
+import findJavaHome from 'find-java-home';
```

Comment:
This import doesn't allow us to call findJavaHome on line 22 since we are importing it as a namespace instead of a callable. So I fixed this.